tasks/eslint.js

```diff
@@ -0,0 +1,16 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+module.exports = function (grunt) {
+  'use strict';
+
+  grunt.config('eslint', {
+    options: {
+      eslintrc: '.eslintrc'
+    },
+    app: [
+      '*/.js', 'bin/*.js', 'tasks/*.js', 'templates/*.js', 'test/**/*.js'
```

Comment:
This needs to be `'*.js'`. This currently won't lint *.js files in the root dir. Once you tweak that, you should get something like `✖ 20 problems (3 errors, 17 warnings)` (mostly semicolon issues in the Gruntfile.js)

```
$ grunt eslint

Running "eslint:app" (eslint) task

Gruntfile.js
   7:6   error    fs is defined but never used  no-unused-vars
   7:25  warning  Extra semicolon               semi
   ...
  86:2   warning  Extra semicolon               semi

mailer.js
  24:4  error  Trailing spaces not allowed  no-trailing-spaces

translator.js
  35:87  error  Strings must use singlequote  quotes

✖ 20 problems (3 errors, 17 warnings)
```